### PR TITLE
Task 4 - ordered list

### PR DESCRIPTION
**Pull request by irinaNMN**
image
It HTML file contains the top level heading and ordered list. The title and a top level heading is Ordered List. I put a second level heading above the list with text describing what types of items the list contains.

### CODE REVIEW

File: ordered.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Ordered List</title>
+</head>
+<body>
+<h1>Ordered List</h1>
+<h2>ETAPELE ACTIVITĂŢII INFRACŢIONALE</h2>
+<di>
```
Comment (ngoncearu) [inline ordered.html]: di tag nu este in HTML

File: ordered.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    <ol>
+        <li> al naşterii sau conceperii ideii de săvîrşire a infracţiunii</li>
+        <li>al deliberării în care persoana cîntăreşte motivele pro şi contra ideii de a săvîrşi o infracţiune.</li>
+        <li>al deciziei sau hotărîrii de a săvîrşi o infracţiune.</li>
+    </ol>
+    </li>
+    <li>
+        Perioada externă <br>
+        <span>Pregătirea de infracţiune sau actele preparatorii este prima fază a perioadei externe a activităţii infracţionale şi constau în anumite acte, activităţi de procurare de date, informaţii ori de adaptare a mijloacelor sau instrumentelor ce vor fi folosite la săvîrşirea infracţiunii sau în activităţi de creare a condiţiilor favorabile săvîrşirii infracţiunii. Aceste acte preparatorii sunt posibile numai la infracţiunile intenţionate.</span>
+      <br> <i><span>Pentru a fi considerate acte de pregătire a infracţiunii activităţile desfăşurate trebuie să îndeplinească anumite condiţii:</span></i>
```
Comment (ngoncearu) [inline ordered.html]: aici span poate fi omis si poti pune textul direct in i tag daca vreai sa fie italic

**Review by ngoncearu — CHANGES_REQUESTED** [review]

File: ordered.html (hunk 1 above)
Comment (irinaNMN) [reply ordered.html]: done!

File: ordered.html (hunk 2 above)
Comment (irinaNMN) [reply ordered.html]: done!

**Review by ngoncearu — APPROVED** [review]

**Pull request merged**